src/work/SimpleDAO.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title SimpleDAO
+ * @dev A simplified version of a DAO that allows members to create proposals,
+ */
+contract SimpleDAO {
```

Comment:
consider adding a commit reveal scheme for this DAO. Since the votes are public there are possibilities of vote manipulation. This involves changing the vote method to be split into a commit vote and reveal vote method. Commit vote will lock in the user's vote without publishing it via event, and the after a specific timestamp allow the reveal vote method to be called and have the event broadcasted then

Reply:
We can use a hash of the user's vote choice plus a secretpassword to ensure the user's vote does not change between committing and revealing

Reply:
Would also be good to have a yes vote threshold to make sure 1 yes doesnt execute a proposal

Reply:
investigating